content/docs/deployment/clients/_index.md

```diff
@@ -1,328 +1,684 @@
+
+4. The repacked MSI will then be available in the `Uploaded Files` tab of the
+   artifact collection.
+
+![](create_msi_uploaded.svg)
+
+**Option 2: Using the command line**
+
+{{% notice note %}}
+
+In this section we'd like to draw your attention to the fact that repacking the
+Windows MSI package can be done on _any_ platform. It does not have to be done
+on Windows.
+
+Also note that in the commands below we have omitted version numbers from the
+file names to make the commands more concise, however it is useful to include
+the version number and architecture tags in your output file names so that you
+know exactly what they are.
 
 {{% /notice %}}
 
-{{% notice note "Winget install is not supported - you should pin/denylist Velociraptor in winget" %}}
+To repack the MSI with a custom config on the command line we use the `config`
+command, and the `repack` subcommand, with the `--msi` flag.
 
-If your windows environment use winget command line tool (or
-`Romanitho/Winget-AutoUpdate` for user toasts) for unattended 3rd party
-app auto updates on windows.  This can lead to failed downgrade
-message issues (as at 5 Feb 2024) or when the winget version numbers
-are fixed unexpected upgrade toasts and server client version
-mismatches
+In all cases we
+need to tell velociraptor which MSI we want to repack (usually it's the
+[official release MSI](/downloads/) - 64-bit or 32-bit) and what the output file
+should be named.
 
-We strongly recommend getting your admins to pin the Velociraptor
-version in winget and deny list upgrading Velociraptor - for
-Winget-autoupdate if your environment uses them - please see
-https://learn.microsoft.com/en-us/windows/package-manager/winget/pinning
-/ https://github.com/Romanitho/Winget-AutoUpdate (search for
-blacklist) for more information.
+{{< tabs >}} {{% tab name="Linux" %}}
+```shell
+./velociraptor config repack --msi velociraptor-windows.msi client.config.yaml velociraptor-windows-repacked.msi
+```
+{{% /tab %}}
+{{% tab name="Windows" %}}
+```shell
+velociraptor.exe config repack --msi velociraptor-windows.msi client.config.yaml velociraptor-windows-repacked.msi
+```
+{{% /tab %}}
+{{% tab name="macOS" %}}
+```shell
+./velociraptor config repack --msi velociraptor-windows.msi client.config.yaml velociraptor-windows-repacked.msi
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+If you are using Velociraptor [organizations](/docs/deployment/orgs/) ("orgs")
+then you can obtain the client configuration file for each org from the Home
+page in the GUI. Ensure that you are in the `root` org so that the configs for
+all orgs are accessible.
+
+![Where to find the client configs for all orgs](client_config_orgs.svg)
+
+{{% notice note "Re-signing the repacked MSI" %}}
+
+While the Velociraptor binary inside the MSI is officially signed by Rapid7 LLC
+and is unaffected by the MSI repacking process, modifying the MSI file will
+invalidate its (separate) digital signature. For this reason we highly recommend
```

Comment:
I dont think we sign the msi any more because the signature will always be invalidated anyway

Reply:
Good point. I wasn't aware of the change. Updated accordingly.